_tests/tests_suite_test.go

```diff
@@ -27,7 +28,7 @@ func init() {
 }
 
 func getRandAppName() string {
-	return fmt.Sprintf("apps-test-%d", rand.Int())
+	return fmt.Sprintf("test-%d", rand.Int())[:24]
```

Comment:
this may panic if `rand.Int()` returns a number with fewer than 19 digits. imo, better to use [`rand.Intn`](https://godoc.org/math/rand#Intn) to limit length